package.json

```diff
@@ -36,6 +36,7 @@
     "@types/redux-mock-store": "0.0.6",
     "@types/sinon": "^1.16.32",
     "chai": "^3.5.0",
+    "cheerio": "^0.22.0",
```

Comment:
Because I need to create an instance of textarea to [textAreaTools specs](https://github.com/MasterLemon2016/LeanMood/pull/90/files#diff-d6efbf14095244cd596afb63e7246c6cR13). Cheerio is used by enzyme under hood to create html wrappers.